### DISCUSSION TITLE
Use unrooted Shizuku without an external wireless network?

### DISCUSSION THREAD

**Discussion by CapoFantasma97** [RikkaApps/Shizuku#611] [Q&A]
Sometimes it happens that I need to restart the phone away from home and there are no public wireless networks available in the area and it's inconvenient and/or not really possible to ask other people to share their hotspot with me for a moment.
Without Shizuku, I wouldn't be able to use the Pixel IMS app to force enable VoLTE, without which calls don't work with my operator.

Is it possible to work around this? I tried even with the hotspot mode out of desperation, but as expected it doesn't interact with wireless debug nor I could connect my phone to itself lol.

**Comment by BatuBey5G** [comment]
You can buy an ESP32 and code it to act like a wifi or root your device 
https://www.instructables.com/Creating-a-Wireless-Network-With-ESP32-AP-Mode/

**Comment by chaoscreater** [comment]
Keen to know as well
